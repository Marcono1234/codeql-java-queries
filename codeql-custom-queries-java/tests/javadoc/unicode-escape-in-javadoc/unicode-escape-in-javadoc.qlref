./javadoc/unicode-escape-in-javadoc.ql
