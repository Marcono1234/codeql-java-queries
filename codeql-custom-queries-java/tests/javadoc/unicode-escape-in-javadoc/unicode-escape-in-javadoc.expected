| Test.java:2:4:2:9 | \\u1234 | Contains Unicode escape `\\u1234` |
| Test.java:3:4:3:9 | \\uabcd | Contains Unicode escape `\\uabcd` |
| Test.java:4:4:4:9 | \\uABCD | Contains Unicode escape `\\uABCD` |
| Test.java:5:4:5:11 | \\uuuABCD | Contains Unicode escape `\\uuuABCD` |
